docs/topics/install-olympia/installation.rst

```diff
@@ -56,9 +57,18 @@ The following packages will get you set for olympia::
 MySQL
 ~~~~~
 
+By default, Olympia uses the root user and no password to connect to the database. If you want to change this, you could either add the database settings in your :ref:`local_settings.py <example-settings>` or set the env variable DATABASE_URL by doing::
```

Comment:
Some nits:

s/uses the root user and not password to connect to the database/connects to the `olympia` database running on `localhost` as the user `root`, with no password/

s/could/can/

s/env/environment/

s/DATABASE_URL/`DATABASE_URL/

s/by doing//